### DISCUSSION TITLE
Phazerville Suite v1.5

### DISCUSSION THREAD

**Discussion by djphazer** [djphazer/O_C-Phazerville#2] [Announcements]
(Hastily prepared on a Sunday night, watch for bugs...)

After trimming some fat, I managed to fit several of the stock O&C firmware apps in alongside the full Hemisphere Suite + all the tweaks!

- The main builds include: **Quantermain, Meta-Q, Acid Curds, Harrington 1200**
- The "+stock" build also includes: **Sequins** & **Quadraturia*** but disables _Captain MIDI, Neural Net, and Enigma_

I also prepared a flipped version and a VOR patched version. Documentation coming soon...

*Known bug: the frequency display in Quadraturia is broken, but still functional

<hr /><em>This discussion was created from the release <a href='https://github.com/djphazer/O_C-BenisphereSuite/releases/tag/PSv1.5'>Phazerville Suite v1.5</a>.</em>

**Comment by PedalEffected** [comment]
This is truly great work! I had recently debated buying a 2nd O&C in order to run the OG firmware and Hemispheres. I like your version a lot - any chance to get **Piqued** added back in? 

Personally, I could live without Meta-Q and Scale Editor if to make room for Piqued in your version. Are you open to making changes?

**Reply by djphazer** [reply]
I'm definitely open to suggestions! Others have requested Piqued as well, so I'll try to make it available in the next release. Out of curiosity, what specific features do you like about that one? I wonder what it can do that can't be achieved with Hemispheres...

**Comment by PedalEffected** [comment]
I would have to double check to truly answer in full why I like it so much. There was a bit of an interaction and settings aspect that was nice, since it was all on one screen.I can get detailed for you tonight but I would say it offered certain more direct settings that Ebb and LFO just didn’t amongst other ones. I would say similar to why people preferred Quantermain over some Hemispheres versions. There’s just a bit of magic with some of those full screen applets.

**Reply by djphazer** [reply]
I get it. _adanoslomry_ on reddit explained:

> The Hemispheres envelopes are very basic by comparison. Piqued has flexible CV control, euclidean patterns, loop/burst features, EOC triggers, and more that are really useful for generative patches.

I ask because I want to try to replicate some of that flexible behavior in simpler, more modular ways, with applets and such. I have this dream of chaining applets together to achieve, for example, an envelope generator with Euclidean patterns - in one hemisphere.

**Comment by PedalEffected** [comment]
I would very much agree with that assessment.It would be worthwhile to bring over and if needed to sacrifice some other less used items.For me, scale editor was weird but also redundant of some other applets. I’d believe there to be room on there for Piqued and a few things worth taking off to make it happen.Thank you for doing this in the first place, it’s amazing work so far

**Comment by vestedaf** [comment]
I installed v1.5 (flipped) last night and I'm loving it so far. One thing I ran into while attempting a long-overdue calibration, is when I got to the cv input steps, adjusting cv1 seemed to affect cv2, and so on. Unfortunately this left me unable to adjust cv1. I'll admit that I've never calibrated my o_c before this, and I don't know if other firmware versions behave this way, so maybe I'm not understanding the functionality.

I figured I'd post here because you seem smart and I wanted to shout out a BIG THANKS @djphazer! :)

**Reply by djphazer** [reply]
I've noticed this too... it only lets you set one calibration that applies to all four ADC inputs. This isn't a huge problem for me, but there is a slight variance of maybe ~0.02V between the inputs. I need to try the stock firmware for calibration, but I doubt it's much different.

**Reply by djphazer** [reply]
Oh! Actually, this is probably a bug I introduced only in the flipped version. I recommend using the non-flipped firmware for calibration!

**Reply by vestedaf** [reply]
Hey! I spent some more time on this last night and got your flipped v1.5 calibrated perfectly! Here's what I found: the cv inputs are flipped, so cv1 affects cv4, cv4 affects cv1, cv2 affects cv3, and cv3 affects cv2. I think you're right, likely introduced by creating the flipped version. That said, I can't be alone in SUPER appreciating your flipped version, so I'll keep it this way for now and hopefully you could sort it out in a future release. Again, HUGE thanks to you. Your work here is awesome!
